### ISSUE TITLE
[Edge] Editor doesn't get focus after dismissing the text alternative balloon

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#716]
## 🐞 Is this a bug report or feature request? (choose one)

- Bug report

## 💻 Version of CKEditor

1.0.0-alpha2

## 📋 Steps to reproduce

1. Go to http://localhost:8125/ckeditor5-core/tests/manual/article.html.
2. Focus the image.
3. Open text alternative balloon.
4. Put the selection in `Heading 1`.
5. Try to type something.

## ✅ Expected result

You should be able to type in `Heading 1`.

## ❎ Actual result

The image toolbar is still visible and there is no possibility to type.

## 📃 Other details that might be useful

**GIF**

![bug_cke5](https://user-images.githubusercontent.com/20988892/33941596-79480a70-e013-11e7-9067-dd9dedcaeaba.gif)

**Notes**

If you try to press <kbd>Delete</kbd>, it will work.

**Other information**

OS: Windows 10
Browser: Edge 16

**Comment by oleq** [comment]
It could somehow (?) be that the input in the text alt balloon retains the focus when the balloon disappears and the focus is not transferred to editable (the editable does not even get the blue outline) but it's actually lost. 

Feels more like a bug in Edge than actual CKE problem but it looks critical to me, making the editor useless after a very common UI interaction. WDYT @Reinmar?

**Comment by Reinmar** [comment]
Uh, sorry, I misread the TC. So it seems to be an Edge bug and yep, it's quite severe.

**Comment by Reinmar** [comment]
Since, I'm not sure what `this._hideForm( true )` is I guess @Mgsy will have hard time finding this out too :D 

Whatever you find, if it's Edge, it must be reported.

**Comment by oleq** [comment]
Just make sure the same patch is also used in ckeditor5-link.

**Comment by Reinmar** [comment]
Since `_hideForm()` is private and always get `true`, can we just remove this param at all?

**Comment by Reinmar** [comment]
BTW, on <kbd>Esc</kbd> the focus should actually move back to the toolbar if it was there.

So, in the future, when working on a11y we'll need to remember which element hold focus before the form was opened and move focus back there. But that's future.

**Comment by FilipTokarski** [comment]
This bug is fixed in the new upcoming Edge v.79.0.309.56.

**Comment by Mgsy** [comment]
Fixed in the new Edge release.
